src/sass/index.sass

```diff
@@ -5,6 +5,9 @@
   display: flex
   flex-direction: column
 
+  &.game
+    justify-content: center
```

Comment:
Vertically aligns all windows to the center

Reply:
neat!